### PR TITLE
Fixes issue #98

### LINKED ISSUE #98: SubjectData builder should allow 'Remove' TransactionType
https://github.com/mdsol/rwslib/blob/1a86bc072d408c009ed1de8bf6e98a1769f54d18/rwslib/builders/clinicaldata.py#L61

Remove should be allowed because Rave Web Services supports it and will inactivate the subject.

### PR DESCRIPTION
**Pull request by anewbigging**
Fixes #98 
@glow-mdsol @isparks please review

### CODE REVIEW

File: rwslib/builders/clinicaldata.py [hunk 1]
```diff
@@ -50,22 +58,31 @@ def build(self, builder):
...
+        subject_key,
+        subject_key_type="SubjectName",
+        transaction_type="Update",
+    ):
         """
         :param str site_location_oid: :class:`SiteLocation` OID
         :param str subject_key: Value for SubjectKey
         :param str subject_key_type: Specifier as to the type of SubjectKey (either **SubjectName** or **SubjectUUID**)
-        :param str transaction_type: Transaction Type for Data (one of **Insert**, **Update**, **Upsert**)
+        :param str transaction_type: Transaction Type for Data (one of **Insert**, **Update**, **Upsert**, **Remove**)
```
Comment (iansparks) [inline rwslib/builders/clinicaldata.py]: And Context?

Comment (anewbigging) [reply rwslib/builders/clinicaldata.py]: Yes, will update the docstrings

File: rwslib/builders/clinicaldata.py [hunk 2]
```diff
@@ -399,7 +399,7 @@ def __init__(
         :param str itemoid: OID for the matching :class:`ItemDef`
         :param str value: Value for the the ItemData
         :param str specify_value: 'If other, specify' value - *Rave specific attribute*
-        :param str transaction_type: Transaction type for the data
+        :param str transaction_type: Transaction Type for Data (one of **Insert**, **Update**, **Upsert**, **, Context**, **Remove**)
```
Comment (glow-mdsol) [inline rwslib/builders/clinicaldata.py]: I think the `**` are in the wrong place here

Comment (anewbigging) [reply rwslib/builders/clinicaldata.py]: gah, thanks.  New push to address.

**Pull request merged**